taxi/tests/test_models.py

```diff
@@ -0,0 +1,55 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+
+from taxi.models import Manufacturer, Car
+
+
+class ModelTests(TestCase):
+    @classmethod
+    def setUpTestData(cls):
+        cls.username = "test_user"
+        cls.license_number = "AAA00001"
+        cls.password = "Password1!"
+        cls.first_name = "James"
+        cls.last_name = "Bond"
+        cls.name = "BMW"
+        cls.country = "Germany"
+        cls.model = "M4"
+
+    def test_manufacturer_str(self):
+        manufacturer = Manufacturer.objects.create(
+            name=self.name,
+            country=self.country,
+        )
+        self.assertEqual(str(manufacturer), f"{self.name} {self.country}")
+
+    def test_driver_str(self):
+        driver = get_user_model().objects.create(
```

Comment:
You are creating a user here but not setting the first_name and last_name. These fields are used in the str method of the user model, so you should set them here to make the test more realistic.